### PR TITLE
Add Elo Formula calculation to utils

### PR DESCRIPTION
**Pull request by metalvegetarianoprogresivo**
## What does this do?
We added a formula to calculate the new values of the ELO of a player's match.

## How can this change be undone in case of failure?

1. Notify the administrators
2. Request a revert of the PR

ping @Xotl

### CODE REVIEW

File: src/utils.js [hunk 1]
```diff
@@ -102,6 +102,16 @@ const removeAlreadyChallengedPlayers = (players, playerId, in_progress) => {
 
 const removeEmptyArray = array => array.filter(i => (i.length === 0) ? false : true)
 
+const eloCalculation = (playerAElo, playerBElo, playerAScore, playerBScore) => {
+    const probabilityOfWinPlayerA = 1.0 * 1.0 / (1 + 1.0 * Math.pow(1, 1.0 * (playerAElo - playerBElo) / 400))
+    const probabilityOfWinPlayerB = 1.0 * 1.0 / (1 + 1.0 * Math.pow(1, 1.0 * (playerBElo - playerAElo) / 400))
+
+    return {
+        playerANewElo: playerAElo + 10 * (playerAScore - probabilityOfWinPlayerA),
```
Comment (Xotl) [inline src/utils.js]: You're fixing the *K* factor to 10. The recommendation is to change that based on how strong/weak the players are. __(see image below)__
image

Comment (metalvegetarianoprogresivo) [reply src/utils.js]: Agree.

Comment (metalvegetarianoprogresivo) [reply src/utils.js]: Do you think we should use the USCF values? `32` for players under 2100, `24` for players between 2100 and 2400 and `16` for player over 2400?

File: src/utils.js [hunk 2]
```diff
@@ -102,6 +102,29 @@ const removeAlreadyChallengedPlayers = (players, playerId, in_progress) => {
...
+        case (elo < 2100):
+            return 32
+        case (elo >= 2100 && elo <= 2400):
+            return 24
+        case (elo > 2400):
+            return 16
+        default:
+            return 32
+    }
+}
```
Comment (Xotl) [inline src/utils.js]: The `factorKCaculator` isn't working properly, check this CodePen:

https://codepen.io/Xotl/pen/MWgNKEO?editors=1012

I added a suggestion named `newFactorKCaculator`.

**Comment by metalvegetarianoprogresivo** [comment]
@Xotl can you review this again? ♻️

**Pull request merged**